lightning/src/ln/channel.rs

```diff
@@ -671,6 +671,12 @@ impl<Signer: Sign> Channel<Signer> {
 
 		let feerate = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::Normal);
 
+		let value_to_self_msat = channel_value_satoshis * 1000 - push_msat;
+		let commitment_tx_fee = Self::commit_tx_fee_msat(feerate, 4);
```

Comment:
Could you add constant for 4 given it is used in a couple places here and throughout the functional tests?